lib/runtime/src/distributed.rs

```diff
@@ -153,17 +160,7 @@ impl DistributedRuntime {
                 }
             }
         } else {
-            // System server HTTP is disabled, but still create the state for metrics
-            // This ensures uptime_seconds metric is always registered
-            let system_status_state = crate::system_status_server::SystemStatusState::new(
-                Arc::new(distributed_runtime.clone()),
-            )?;
-
-            // Initialize the start time for uptime tracking
-            if let Err(e) = system_status_state.initialize_start_time() {
-                tracing::warn!("Failed to initialize system status start time: {}", e);
-            }
-
+            // System server HTTP is disabled, but uptime metrics are still being tracked via SystemHealth
```

Comment:
Does it make sense to add a small test case that basically does this?
```bash
DYN_SYSTEM_ENABLED=false
assert <request to system status server fails as expected because it's not running>
```

This will be a signal to other developers that something is wrong if they add the server back in this `else` block in future, and catch subtle things like if we change the mechanism or name of the environment variable for toggling support or something.